wallet/core/src/deterministic.rs

```diff
@@ -56,7 +56,7 @@ impl std::fmt::Display for AccountId {
     }
 }
 
-seal! { 0xa7a4, {
+seal! { 0x544d, {
```

Comment:
Since it only works with version 1.77 we have to introduce msrv equals 1.77 in manifest files

Reply:
MSRV added: https://github.com/kaspanet/rusty-kaspa/pull/442/commits/ad701e1c8135bb2846e3b7baad88dd6a64c2b03f